### PR TITLE
[OpenStack] add check for attaching volume

### PR DESCRIPTION
**Pull request by kivio**
add check to make sure volume is attached.

Currently without timeout (but i work on universal solution for all checks in OpenStack driver)

### CODE REVIEW

File: perfkitbenchmarker/openstack/os_disk.py [hunk 1]
```diff
@@ -90,6 +90,14 @@ def Attach(self, vm):
         self.attach_id = result.id
         self.device = "/dev/disk/by-id/virtio-" + result.id[:20]
 
+        is_unattached = True
+        while is_unattached:
```
Comment (cmccoy) [inline perfkitbenchmarker/openstack/os_disk.py]: Could you set an upper bound on the wait time here?

**Comment by voellm** [comment]
I plan to roll the next release today.  Is this ready to integrate?

**Comment by voellm** [comment]
@kivio - Should we target this for v0.23.0 ?  What is left here?

**Comment by kivio** [comment]
@voellm please merge, timeout will be merged in other issue for whole OpenStack driver.

**Pull request merged**